.github/workflows/tests.yaml

```diff
@@ -22,10 +22,11 @@ jobs:
             matrix:
                 php: ['8.1']
                 path:
-                    - tests
-                    - rules-tests
-                    - packages-tests
-                    - utils-tests
+                    - "tests/"
```

Comment:
The extra slashes will break the required name patter to require jobs to pass

Reply:
not sure what you mean?

Reply:
This is now a new job, with extra / in the end, that is not checked by required CI jobs

![Screenshot from 2023-05-12 16-33-45](https://github.com/rectorphp/rector-src/assets/924196/c18219d6-061a-46ab-b2e4-cc9d9801c824)

Reply:
```suggestion
                    - "tests"
```

Reply:
if you mean the github.com required action rules: I think you need to re-configure them no matter I leave the `/` or not?

but I am fine dropping them. I like trailling slahes because they convey the notion of a file path

Reply:
you also need to re-confiure the rules for new the jobs added, right?

Reply:
I have no strong opinion. just tell me what needs to be done so we can merge it :)

Reply:
It's adds us work for no gain :)

Please revert like suggested: https://github.com/rectorphp/rector-src/pull/3815#discussion_r1192459132